x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -70,6 +72,25 @@ const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
   );
 };
 
+const PolicyLink: React.FC<{ name: string; route: string }> = ({ name, route }) => {
+  const history = useHistory();
+
+  return (
+    <EuiLink
+      onClick={(event: React.MouseEvent) => {
+        event.preventDefault();
+        history.push(route);
+      }}
+    >
+      {name}
+    </EuiLink>
+  );
+};
+
+const renderPolicyNameLink = (value: string, _item: PolicyData) => {
```

Comment:
i noticed this in Paul's code too, what does the `_item` do?

Reply:
it gives you access to the all of the data that the table columns are based on, in this case the `PolicyData`

Reply:
to expand on @parkiino 's comment. The confusion might be in that in my commit I was actually NOT using the `item` data (thus why I named it with a `_`), but needed to define the parameter in order to get around a TS error - [thread here (slack EUI channel) for reference](https://elastic.slack.com/archives/C7QC1JV6F/p1581094308212600?thread_ts=1581092098.211600&cid=C7QC1JV6F)

Reply:
@kevinlog  do you remember when i had that type error on the endpoint details pr? Is this another way to solve that?